lib/src/components/Store.ts

```diff
@@ -1,11 +1,19 @@
 import { ComponentProvider } from 'react-native';
+import { IWrappedComponent } from './ComponentWrapper';
 
 export class Store {
   private componentsByName: Record<string, ComponentProvider> = {};
   private propsById: Record<string, any> = {};
+  private componentsInstancesById: Record<string, IWrappedComponent> = {};
 
   setPropsForId(componentId: string, props: any) {
     this.propsById[componentId] = props;
+
+    const component = this.componentsInstancesById[componentId];
```

Comment:
Sorry for nitpicking - lets remove the empty lines around this line. I don't think they serve any purpose :)

Reply:
nitpicking is good!